scripts/svgr-configs/logos.ts

```diff
@@ -70,8 +70,18 @@ export const LogosSvgReactOutPutConfig: Options = {
             },
           },
         },
+        {
+          name: 'prefixIds',
+          params: { prefix: Math.random().toString(16) },
```

Comment:
Let's use Nodejs Crypto for this.what do you think?

Reply:
there might be a chance that this will generate duplicated strings

Reply:
```

import crypto from 'crypto'
const hash = crypto.randomBytes(16).toString('hex')
```

Reply:
This approach doesn't help us fixing the ids issue:
https://github.com/svg/svgo/issues/674#issuecomment-285761453